### PR TITLE
Connect swiping solution into rn playground

### PR DESCRIPTION
**Pull request by lukewalczak**
ezgif com-video-to-gif 5
ezgif com-video-to-gif 2

### CODE REVIEW

File: ios/RNKiwiMobile/RNKiwiViewController.m [hunk 1]
```diff
@@ -61,6 +61,10 @@ - (void)viewDidDisappear:(BOOL)animated {
   }
 }
 
+- (void)viewWillAppear:(BOOL)animated {
+  self.navigationController.navigationBar.hidden = YES;
```
Comment (ferrannp) [inline ios/RNKiwiMobile/RNKiwiViewController.m]: Let's remove it from here.

File: ios/reactNativeApp.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -489,45 +489,45 @@
...
-                "${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/Zocial.ttf",
-            );
-            name = "Remove extraneous assets";
-            outputPaths = (
-            );
-            runOnlyForDeploymentPostprocessing = 0;
-            shellPath = /bin/sh;
-            shellScript = $SRCROOT/../scripts/removeExtraneousAssets.sh;
-        };
+			isa = PBXShellScriptBuildPhase;
```
Comment (ferrannp) [inline ios/reactNativeApp.xcodeproj/project.pbxproj]: Let's revert this file.

File: ios/RNNativePlayground/ViewController.m [hunk 3]
```diff
@@ -26,20 +27,21 @@ - (IBAction)showHotelsView:(id)sender {
...
 - (NSDictionary<NSString *, NSObject *> *)initialProperties {
   return @{
-    @"coordinates": @{
-      @"latitude" : @59.9139,
-      @"longitude": @10.7522
-    },
-    @"language": @"en",
-    @"currency": @"EUR"
-  };
+           @"coordinates": @{
```
Comment (ferrannp) [inline ios/RNNativePlayground/ViewController.m]: This needs to be reverted too.

**Comment by mrtnzlml** [comment]
Check the native header while going back from RN. It's not visible and it appears after the final transition is complete. Is it possible to fix even this?

**Pull request merged**